src/components/SavingMessage.css

```diff
@@ -0,0 +1,59 @@
+.blob-wrapper {
+  display: flex;
+  position: absolute;
+  top: -120px;
+  right: 0;
+}
+
+.saving {
+  animation: savingAnimation 2s infinite linear;
+}
+
+.blob-wrapper span {
+  padding-left: 0;
+  position: relative;
+  left: -10px;
+  color: #000;
+  margin: 10px;
+}
+
+.pulse-animation {
```

Comment:
This animation would be in the Style Guide like a type pulse animation, see: https://github.com/FromDoppler/doppler-style-guide/pull/224#discussion_r1116403440

Reply:
Talking with @GusBaamonde the best way was remove the animation in https://github.com/FromDoppler/doppler-style-guide/pull/224#discussion_r1116403440